subcommand_plugin_examples/c_subcommand_plugin_tutorial/pyproject.toml

```diff
@@ -1,16 +1,13 @@
 [build-system]
-requires = ["setuptools>=61.0", "setuptools-scm"]
+requires = ["setuptools>=61.0", "setuptools-scm", "cffi>=1.0.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "temp-converter"
 version = "1.0"
 description = "A custom subcommand written in C that converts Celsius to Fahrenheit"
 requires-python = ">=3.7"
-dependencies = ["conda"]
-
-[tools.setuptools]
-py_modules=["temp-converter"]
```

Comment:
Didn't seem to like putting `packages=` here, so it went into `setup.py`

Reply:
Probably it should be `c_subcommand_plugin_tutorial/temp-converter/pyproject.toml` so that each small project has the same level of nesting, so far I've pulled these up one level.